pkg/flowaggregator/exporter/ipfix.go

```diff
@@ -98,6 +129,7 @@ func NewIPFIXExporter(
 		registry:                   registry,
 		clusterUUID:                clusterUUID,
 		maxIPFIXMsgSize:            int(opt.Config.FlowCollector.MaxIPFIXMsgSize),
+		tls:                        newIPFIXExporterTLSConfig(opt.Config.FlowCollector.TLS),
```

Comment:
I only see it's configured in initialization, how do we handle certs/Secrets update?

Reply:
See comment below in `initExportingProcess`:
```
// We reload the certificate data every time, in case the files have been updated.
```
The only time we need to worry about cert update is when we (re)connect to the server. And a certificate update doesn't mean that we automatically have to force a reconnection.